src/main/java/org/traccar/protocol/TopinProtocolDecoder.java

```diff
@@ -122,13 +150,7 @@ protected Object decode(
             position.set(Position.KEY_VERSION_FW, firmware);
 
             ByteBuf content = Unpooled.buffer();
```

Comment:
If content is not required here, please remove it and allow null value in the `sendResponse` method.

Reply:
The content is needed in this place as the interval is written to it and send back to the tracker. It is just not listed in this extracted view. The content is listed in line 153!